bitnami/fluentd/README.md

```diff
@@ -68,6 +68,9 @@ The following tables lists the configurable parameters of the fluentd chart and
 | `forwarder.securityContext.enabled`             | Enable security context for forwarder pods                                                                     | `true`                                                                                                  |
 | `forwarder.securityContext.fsGroup`             | Group ID for forwarder's containers filesystem                                                                 | `0`                                                                                                     |
 | `forwarder.securityContext.runAsUser`           | User ID for forwarder's containers                                                                             | `0`                                                                                                     |
+| `forwarder.securityContext.*`                   | Other pod security context to be included as-is in pod spec                                                    | `{}`                                                                                                    |
+| `forwarder.containerSecurityContext.enabled`    | Enable security context for the forwarder container                                                            | `true`                                                                                                  |
+| `forwarder.containerSecurityContext.*`          | Other container security context to be included as-is in pod spec                                              | `{ "privileged" : false, "allowPrivilegeEscalation": false, "capabilities": { "drop": ["ALL"] } }`      |
```

Comment:
I have my concerns with this. If we create a custom values.yaml file like this

```
cat myvalues.yaml

containerSecurityContext:
  capabilities:
    add: ["CHOWN", "SETUID". "SETGID"]
```

The chart would take the default `drop` section as well (because it is in the default values.yaml file). Would that cause conflicts?

Reply:
I'm not sure I understand your concern correctly, but with `capabilities` but it is a common pattern that you `drop: ["ALL"]` and then add back only ones you need, so it shouldn't cause a conflict between the `add` and the `drop` when Helm merges them since these are processed by the underlying runtime, e.g Docker/containerd. e.g https://github.com/bitnami/charts/blob/01774ba8cd5984a9db27b0d3c411f398146ccc75/bitnami/nginx-ingress-controller/templates/controller-deployment.yaml#L95-L100

Since the capability "directives" (add/drop) are in dictionaries, Helm will merge fine. e.g

The below works fine and pods start up OK.
```yaml
forwarder:
  daemonUser: fluentd
  daemonGroup: fluentd
  securityContext:
    runAsUser: 1001
    fsGroup: 1001

  containerSecurityContext:
    capabilities:
      add: [ "CHOWN", "SETUID", "SETGID" ]
```
![image](https://user-images.githubusercontent.com/29788154/92380957-93e7df80-f13c-11ea-94fc-77c1ece8bc57.png)


But since the capabilities themselves are an array, which are an override with helm, you can also effectively disable the capability dropping entirely if necessary with a custom values file like:

```yaml
forwarder:
  containerSecurityContext:
    capabilities:
      drop: []
```

Reply:
Great! That's solves my concern